extensions-contrib/opentelemetry-emitter/src/main/java/org/apache/druid/emitter/opentelemetry/OpenTelemetryEmitter.java

```diff
@@ -94,6 +94,7 @@ private void emitQueryTimeEvent(ServiceMetricEvent event)
                        .filter(entry -> !TRACEPARENT_PROPAGATION_FIELDS.contains(entry.getKey()))
                        .forEach(entry -> span.setAttribute(entry.getKey(), entry.getValue().toString()));
 
+      span.setAttribute("exporter.kafka", "exclude");
```

Comment:
You'll need to make this configurable via the properties file before you contribute this upstream.

Reply:
My point was that we'll use this only in our code, not upstream
But yeah, let's make it configurable via properties file